src/libraries/System.Threading.RateLimiting/src/System/Threading/RateLimiting/RateLimiterHelper.cs

```diff
@@ -0,0 +1,26 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+
+namespace System.Threading.RateLimiting
+{
+    internal static class RateLimiterHelper
+    {
+        internal static readonly double TickFrequency = (double)TimeSpan.TicksPerSecond / Stopwatch.Frequency;
```

Comment:
@stephentoub would you think it's a good idea to public the tick frequency in the `Stopwatch`

https://github.com/dotnet/runtime/blob/b199965ab77731ec2269b80a304f2091caa33ba7/src/libraries/System.Private.CoreLib/src/System/Diagnostics/Stopwatch.cs#L27

Reply:
What would you use it for, other than GetElapsedTime, which is already public?

Reply:
Both of those look they're just GetElapsedTime without wrapping in a TimeSpan. Doesn't seem worthwhile.